TrajectoryPlannerROS:
  max_vel_x: 0.75
  min_vel_x: 0.01
  max_vel_theta: 0.5
  min_vel_theta: -0.5
  min_in_place_vel_theta: 0.5

  acc_lim_theta: 0.25
  acc_lim_x: 1.5
  acc_lim_Y: 1.5

  holonomic_robot: false
  sim_time: 5
  
  meter_scoring: true
  heading_lookahead: 0.25
  xy_goal_tolerance: 0.15
  yaw_goal_tolerance: 0.25
